install-dependencies.sh

```diff
@@ -115,73 +119,74 @@ sudo_or_su() {
      glew-devel libicu_devel libpng16-devel libSDL2-devel libsdl2_gfx-devel \
-     libsdl2_image-devel libsdl2_mixer-devel libsdl2_ttf-devel python zlib-devel
+     libsdl2_image-devel libsdl2_mixer-devel libsdl2_ttf-devel python3 zlib-devel libminizip-devel
 
 elif [ "$DISTRO" = "slackware" ]; then
    echo "Please contribute the command and package list for Slackware"
    echo "so that we can add support for it"
 
 elif [ "$DISTRO" = "mageia" ]; then
    echo "Installing dependencies for Mageia..."
-   sudo_or_su urpmi gcc gcc-c++ binutils make asio-devel SDL_image-devel \
+   sudo_or_su urpmi $@ gcc gcc-c++ binutils make asio-devel SDL_image-devel \
     SDL_ttf-devel SDL_mixer-devel png-devel gettext-devel cmake SDL_gfx-devel \
-    jpeg-devel tiff-devel git glew-devel
+    jpeg-devel tiff-devel git glew-devel libminizip-devel
 
 elif [ "$DISTRO" = "debian" ]; then
    echo "Installing dependencies for Debian/Ubuntu Linux, Linux Mint..."
-   sudo apt install git cmake g++ gcc gettext libasio-dev libglew-dev libpng-dev libsdl2-dev \
-    libsdl2-image-dev libsdl2-mixer-dev libsdl2-ttf-dev python zlib1g-dev
+   sudo apt install $@ git cmake g++ gcc gettext libasio-dev libglew-dev libpng-dev libsdl2-dev \
+    libsdl2-image-dev libsdl2-mixer-dev libsdl2-ttf-dev python3 zlib1g-dev libminizip-dev
 
 elif [ "$DISTRO" = "freebsd" ]; then
    echo "Installing dependencies for FreeBSD..."
-   sudo_or_su pkg install git asio cmake gettext glew png sdl2_image sdl2_mixer sdl2_net sdl2_ttf
+   sudo_or_su pkg install $@ git asio cmake gettext glew png sdl2_image sdl2_mixer sdl2_ttf minizip
 
 elif [ "$DISTRO" = "openbsd" ]; then
    echo "Installing dependencies for OpenBSD..."
-   doas pkg_add git cmake gcc g++ gettext-tools glew icu4c libexecinfo png \
-    sdl2-image sdl2-mixer sdl2-net sdl2-ttf
+   doas pkg_add $@ git cmake gcc g++ gettext-tools glew icu4c libexecinfo png \
+    sdl2-image sdl2-mixer sdl2-ttf minizip
    asio_not_packaged "OpenBSD" "doas" || exit 1
 
 elif [ "$DISTRO" = "msys32" ]; then
    echo "Installing dependencies for 32-bit Windows..."
-   pacman -S pacman -S mingw-w64-i686-toolchain git mingw-w64-i686-cmake \
+   pacman -S $@ mingw-w64-i686-toolchain git mingw-w64-i686-cmake \
     mingw-w64-i686-ninja mingw-w64-i686-asio mingw-w64-i686-SDL2_ttf \
     mingw-w64-i686-SDL2_mixer mingw-w64-i686-SDL2_image \
     mingw-w64-i686-glbinding mingw-w64-i686-glew mingw-w64-i686-icu
 
 elif [ "$DISTRO" = "msys64" ]; then
    echo "Installing dependencies for 64-bit Windows..."
-   pacman -S mingw-w64-x86_64-toolchain git mingw-w64-x86_64-cmake \
+   pacman -S $@ mingw-w64-x86_64-toolchain git mingw-w64-x86_64-cmake \
     mingw-w64-x86_64-ninja mingw-w64-x86_64-asio mingw-w64-x86_64-SDL2_ttf \
     mingw-w64-x86_64-SDL2_mixer mingw-w64-x86_64-SDL2_image \
-    mingw-w64-x86_64-glbinding mingw-w64-x86_64-glew mingw-w64-i686-icu
+    mingw-w64-x86_64-glbinding mingw-w64-x86_64-glew mingw-w64-x86_64-icu
```

Comment:
Could these be unified like it was in the workflow file? Maybe define a shell function with $ARCH as argument?

Reply:
Thank you. Also for picking this up. :)